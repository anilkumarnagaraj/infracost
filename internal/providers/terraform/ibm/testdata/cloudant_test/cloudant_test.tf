terraform {
  required_providers {
    ibm = {
      source = "IBM-Cloud/ibm"
      version = "1.42.0-beta0"
    }
  }
}

provider "ibm" {
  # Configuration options
}

resource "ibm_cloudant" "ibm_cloudant" {
  name     = "cloudant-service-name"
  location = "us-south"
  plan     = "standard"

  legacy_credentials  = true
  include_data_events = false
  capacity            = 1
  enable_cors         = true

  cors_config {
    allow_credentials = false
    origins           = ["https://example.com"]
  }

  timeouts {
    create = "15m"
    update = "15m"
    delete = "15m"
  }
}


resource "ibm_cloudant" "cloudant_lite" {
  name     = "cloudant-service-name"
  location = "us-south"
  plan     = "lite"

  legacy_credentials  = true
  include_data_events = false
  capacity            = 1
  enable_cors         = true

  cors_config {
    allow_credentials = false
    origins           = ["https://example.com"]
  }

  timeouts {
    create = "15m"
    update = "15m"
    delete = "15m"
  }
}