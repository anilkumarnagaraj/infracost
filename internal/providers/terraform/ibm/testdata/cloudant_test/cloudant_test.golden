
 Name                                               Monthly Qty  Unit            Monthly Cost 
                                                                                              
 ibm_cloudant.cloudant_standard                                                               
 ├─ Read capacity (100 Reads/sec)                             1  months                 $0.00 
 ├─ Write capacity (50 writes/sec)                            1  months                 $0.00 
 ├─ Global query capacity (5 global queries/sec)              1  months                 $0.00 
 └─ Additional consumption-based charges          Monthly cost depends on usage: $0.00 per GB 
                                                                                              
 ibm_cloudant.ibm_cloudant                                                                    
 ├─ Read capacity (100 Reads/sec)                             1  months                 $0.00 
 ├─ Write capacity (50 writes/sec)                            1  months                 $0.00 
 ├─ Global query capacity (5 global queries/sec)              1  months                 $0.00 
 └─ Additional consumption-based charges          Monthly cost depends on usage: $0.00 per GB 
                                                                                              
 OVERALL TOTAL                                                                          $0.00 
──────────────────────────────────
3 cloud resources were detected:
∙ 2 were estimated, all of which include usage-based costs, see https://infracost.io/usage-file
∙ 1 was free:
  ∙ 1 x ibm_cloudant
Logs:

level=warning msg="No products found for ibm_cloudant.cloudant_standard Additional consumption-based charges, using 0.00"
level=warning msg="No products found for ibm_cloudant.cloudant_standard Global query capacity (5 global queries/sec), using 0.00"
level=warning msg="No products found for ibm_cloudant.cloudant_standard Read capacity (100 Reads/sec), using 0.00"
level=warning msg="No products found for ibm_cloudant.cloudant_standard Write capacity (50 writes/sec), using 0.00"
level=warning msg="No products found for ibm_cloudant.ibm_cloudant Additional consumption-based charges, using 0.00"
level=warning msg="No products found for ibm_cloudant.ibm_cloudant Global query capacity (5 global queries/sec), using 0.00"
level=warning msg="No products found for ibm_cloudant.ibm_cloudant Read capacity (100 Reads/sec), using 0.00"
level=warning msg="No products found for ibm_cloudant.ibm_cloudant Write capacity (50 writes/sec), using 0.00"